src/components/HomeContainer/SuceessStories/SuccessStories.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC } from "react";
+import typography from "../../utility/Typography/Typography.module.css";
+import styles from "./SuccessStories.module.css";
+
+import avatar_Kady from "../../../assets/avatar-kady.jpg";
+import avatar_Aisha from "../../../assets/avatar-aiysha.jpg";
+import avatar_Arthur from "../../../assets/avatar-arthur.jpg";
+
+
+
+const SuccessStory = ({ quote, name, title, avatar, altText }) => {
```

Comment:
Components should no longer handle the typography designs
Here is the suggestion:
`const SuccessStory = ({ quote, personName, title, avatar, altText }) => {
  return (
    <div className={styles.SuccessStories_content}>
      <Typography elType="body2">"{quote}"</Typography>
      <Typography elType="h3Large" className={styles.SuccessStories_name}>
        {personName}
      </Typography>
      <Typography elType="body3" className={styles.SuccessStories_caption}>
        {title}
      </Typography>
      <div className={styles.SuccessStories_image}>
        <img src={avatar} alt={altText} />
      </div>
    </div>
  );
};`